lib/diagnostic.rb

```diff
@@ -8,21 +8,21 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = ruby ./lib/example.rb
```

Comment:
This will work only if the `lib` directory exists in your root directory. You can leave off the `./` if you are already in the directory that `lib` is in.